newswires/client/src/SideNav.tsx

```diff
@@ -59,48 +105,129 @@ export const SideNav = () => {
 				}
 				onClose={() => setNavIsOpen(false)}
 			>
-				<div>
-					<SearchGroup title="Agencies" items={agencies} />
-					<SearchGroup title="Saved searches" items={savedSearches} />
-					<SearchGroup
-						title="Search history"
-						items={searchHistoryItems}
-						isEmptyMessage="No search history available yet"
-					/>
-				</div>
+				<EuiCollapsibleNavGroup title={'Suppliers'}>
+					<EuiListGroup
+						maxWidth="none"
+						color="subdued"
+						gutterSize="none"
+						size="s"
+					>
+						{supplierItems.map(({ label, colour, isActive, onClick }) => {
+							return (
+								<EuiListGroupItem
+									key={label}
+									label={label}
+									onClick={onClick}
+									icon={
+										<div
+											css={css`
+												width: 0.5rem;
+												height: 1.5rem;
+												background-color: ${isActive ? colour : 'transparent'};
+											`}
+										/>
+									}
+									aria-current={isActive}
+								/>
+							);
+						})}
+					</EuiListGroup>
+				</EuiCollapsibleNavGroup>
+				<EuiCollapsibleNavGroup title={'Search history'}>
+					{searchHistoryItems.length === 0 ? (
+						<EuiText size="s">{'No history yet'}</EuiText>
+					) : (
+						<EuiBadgeGroup color="subdued" gutterSize="s">
+							{searchHistoryItems.map(({ label, query, resultsCount }) => {
+								return (
+									<EuiBadge
+										key={label}
+										color="secondary"
+										onClick={() => {
+											handleEnterQuery(query);
+										}}
+										onClickAriaLabel="Apply filters"
+									>
+										{label}{' '}
+										<EuiBadge color="hollow">
+											{resultsCount === 30 ? '30+' : resultsCount}
+										</EuiBadge>
+									</EuiBadge>
+								);
+							})}
+						</EuiBadgeGroup>
+					)}
+				</EuiCollapsibleNavGroup>
 			</EuiCollapsibleNav>
 		</>
 	);
 };
 
-const SearchGroup = ({
+/** 
```

Comment:
I added the 'singleSearchAsListOfBadges' component in an earlier draft of this branch, where it looked like we were going to add more in-depth support for keyword searches. It seems v likely that we'll add this back in in a future PR, so hopefully this will avoid reinventing the wheel later